cabal.project

```diff
@@ -42,7 +42,7 @@ package io-classes
   flags: +asserts
 
 package strict-stm
-  flags: +asserts
+  flags: +asserts +checktvarinvariant
```

Comment:
should this be 

```suggestion
  flags: +asserts +checktvarinvariants
```

Reply:
In that commit we're still using `strict-stm` which used `checktvarinvariant`; unfortunatelly the `strict-checkd-vars` package renamed the flag to `checktvarinvariants`.